paparazzi-gradle-plugin/src/test/java/app/cash/paparazzi/gradle/PaparazziPluginTest.kt

```diff
@@ -1396,6 +1403,11 @@ class PaparazziPluginTest {
     var generatedSettings = false
     var generatedGradleProperties = false
 
+    File(projectRoot, "build-cache").delete()
+    projectRoot.walkTopDown()
+      .filter { it.name == "build" && it.isDirectory }
+      .forEach { it.deleteRecursively() }
+
```

Comment:
Without this I had super flaky tests.

Reply:
can you elaborate some more, as I haven't noticed this in my local dev

Reply:
It would get issues like tasks being up-to-date when running the test the second time since the build directory already existed.

Reply:
... https://github.com/cashapp/paparazzi/pull/695